Firestore/Source/Local/FSTLRUGarbageCollector.h

```diff
@@ -103,4 +105,6 @@ extern const firebase::firestore::model::ListenSequenceNumber kFSTListenSequence
 - (int)removeOrphanedDocumentsThroughSequenceNumber:
     (firebase::firestore::model::ListenSequenceNumber)sequenceNumber;
 
+- (size_t)onDiskSize;
```

Comment:
Could this name be the same as `byteSize` above?

Reply:
Done.